source/controlTypes.py

```diff
@@ -662,21 +664,46 @@ class OutputReason(Enum):
 REASON_CARET = OutputReason.CARET
 REASON_ONLYCACHE = OutputReason.ONLYCACHE
 
+
+class IS_CURRENT(Enum):
+	"""Values to use within NVDA to denote 'current' values.
+	These describe if an item is the current item within a particular kind of selection.
+	EG aria-current
+	"""
+	NO = "false"
+	Yes = "true"
+	PAGE = "page"
+	STEP = "step"
+	LOCATION = "location"
+	DATE = "date"
+	TIME = "time"
+
+	def getTranslationForUI(self):

```

Comment:
Just a thought, if you'd make this:
```suggestion
	def __str__(self):
```

Then, you could do str(value), which is a lot shorter and I think if done consistently (e.g. for roles and states in the future), it is more straight forward.
May be you find the current method more obvious, though.

Reply:
I think this might cause some confusion, I want it to be clear that the returned string is translated, and not just a `str` representation of the enum.